src/core/unittest/SettingsTest.cpp

```diff
@@ -417,3 +417,69 @@ TEST(SettingsTest, GlobalSettingsSizesSet)
                 &InternalSettings));
     }
 }
+
+TEST(SettingsTest, GlobalRawDataPathProcsSetAndGet)
+{
+    uint16_t SetCpus[] = {0};
```

Comment:
We always run on at least 2 (more likely min 4) CPU machines, so feel free to expand this to at least 2.